kafka-connect-upload-source/src/main/java/org/radarbase/connect/upload/converter/altoida/summary/AltoidaSummaryProcessor.kt

```diff
@@ -33,7 +32,7 @@ class AltoidaSummaryProcessor : StatelessCsvLineProcessor() {
                     line.getValue("GENDER").toInt().toGender(),
                     line.getValue("CLASS").toInt().classify(),
                     line.getValue("NMI").toDouble(),
-                    line.getValue("GROUNDTRUTH").toInt().toGroundTruth())
+                    GroundTruth.UNKNOWN)
```

Comment:
@nivemaham I also made this `UNKNOWN` or would it be better to remove this from the schema?